tests/test_compliance/test_compliance_suite.py

```diff
@@ -131,8 +133,8 @@ def map_object(
     source_object: dict[str, Any],
     expected_target_object: dict[str, Any],
     source_sv: SchemaView,
-    invertible: bool = False,
-    index: bool = False,
+    invertible: bool = False,  # noqa: FBT001, FBT002
+    index: bool = False,  # noqa: FBT001, FBT002
```

Comment:
one of the linters gets in a tizzy if you have boolean arguments, so turning off warnings here

Reply:
I'm guessing that it wants to see them as keyword-only arguments? I do like that style, but I'm not opposed to ignoring it for now.